scout/adapter/mongo/panel.py

```diff
@@ -235,7 +235,7 @@ def gene_panel(self, panel_id, version=None):
 
         return None
 
-    def gene_panels(self, panel_id=None, institute_id=None, version=None):
+    def gene_panels(self, panel_id=None, institute_id=None, version=None, include_hidden=False):
```

Comment:
This change has consequences also on the command line. When for instance a user removes a panel in the app, if you `run scout --demo view panels`, you get this:

![image](https://user-images.githubusercontent.com/28093618/137727921-84ad3036-f663-4aaf-8538-715a945a01b0.png)

If you try to export the panel with `scout --demo export panel panel1` the export of the panel genes works. 

It think it would be best if this function has include_hidden=True as default, so that all panels will be visible from the command line (the command line is generally used by system admins)

Reply:
Good point. I have added another column that shows if a panel is hidden or not.